### ISSUE TITLE
(orignal theme compatibility patch) Do not color arithmetical operators

### ISSUE THREAD

**Issue by Trismegistos84** [deplorableword/textmate-solarized#61]
Solarized Dark and Light themes orginally does not color aritchmetical operators e.g. (=, +, -, ++, +=, &&, ||) for any language e.g.

python ![python](http://ethanschoonover.com/solarized/img/screen-python-dark.png)

java ![java](http://ethanschoonover.com/solarized/img/screen-java-dark.png)

Sublime Text 3 port follows these setting in some languages e.g. C++, Haskell but for other languages like Python or Ruby it colors these operators same as keywords. This is deviation from orginal theme and additionally it introduces noise.

I have prepared lines that has to be added to make Sublime scheme compatible with orginal one.
### Add following lines to dark scheme:

```xml
<dict>
    <key>name</key>
    <string>Arithmetical, Assignment, Comparision Operators</string>
    <key>scope</key>
    <string>keyword.operator.comparison, keyword.operator.assignment, keyword.operator.arithmetic</string>
    <key>settings</key>
    <dict>
        <key>fontStyle</key>
        <string></string>
        <key>foreground</key>
        <string>#839496</string>
    </dict>
</dict>
```

### Add following lines to light scheme:

```xml
<dict>
    <key>name</key>
    <string>Arithmetical, Assignment, Comparision Operators</string>
    <key>scope</key>
    <string>keyword.operator.comparison, keyword.operator.assignment, keyword.operator.arithmetic</string>
    <key>settings</key>
    <dict>
        <key>fontStyle</key>
        <string></string>
        <key>foreground</key>
        <string>#657B83</string>
    </dict>
</dict>
```

**Comment by deplorableword** [comment]
Hey,

Sounds good :+1: 

Could you could hit Edit on Github for Dark https://github.com/deplorableword/textmate-solarized/edit/master/Solarized%20(dark).tmTheme and Light https://github.com/deplorableword/textmate-solarized/edit/master/Solarized%20(light).tmTheme paste / replace the lines which need replacing and hit Save.

That should generated a pull request, which I can automatically merge in via GitHub.

Thanks

**Comment by Trismegistos84** [comment]
@deplorableword The change was merged few days ago. I didn't know I am responsible for closing this issue.

**Comment by jibsen** [comment]
I think it was merged over in the [SublimeColors repo](http://github.com/SublimeColors/Solarized/issues/37), and not this one, which is for the TextMate version.

**Comment by Trismegistos84** [comment]
@jibsen  You are right. This is mistake on my side. I have initially created this issue believing this is bug tracker for Sublime text. I could merge the change but I am not able to check it because I do not have Mac nor TextMate.
